admin/test/test_images.py

```diff
@@ -116,3 +167,359 @@ def test_multiple_ami(self):
         item dictionary.
         """
         self.assert_packer_amis(PACKER_OUTPUT_US_ALL)
+
+
+class PackerConfigureTests(TestCase):
+    """
+    Tests for ``PackerConfigure``.
+    """
+    def setUp(self):
+        """
+        Create a temporary working directory.
+        """
+        super(PackerConfigureTests, self).setUp()
+        self.working_directory = FilePath(self.mktemp())
+        self.working_directory.makedirs()
```

Comment:
:question: in `flocker/testtools/_base _MktempMixin` we have some helpers for this, any chance they would work for you?